easybuild/easyconfigs/o/OpenCV/OpenCV-4.2.0-foss-2019b-Python-3.7.4.eb

```
@@ -0,0 +1,46 @@
+name = 'OpenCV'
+version = '4.2.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://opencv.org/'
```

The homepage URL can be switched to https
```suggestion
homepage = 'https://opencv.org/'
```

---

@boegel since this is such a minor change, I'll commit it myself to spare you the hassle :slightly_smiling_face: